CONTRIBUTING.rst

```diff
@@ -0,0 +1,109 @@
+======================
+Contributing to DWI_ML
+======================
+
+Welcome to DWI_ML ! We're excited you're here and want to contribute.
+
+This article documents how to contribute improvements to DWI_ML.
+
+The development process
+=======================
+
+When writing code, please pay attention to the following aspects:
+
+Coverage
+--------
+
+We use `pytest`_ to write the tests of the code.
+
+If you are adding code into a module that already has a test file, add
+additional tests into the respective file.
+
+New contributions are required to have as close to 100% code coverage as
+possible. This means that the tests written cause each and every statement in
+the code to be executed, covering corner-cases, error-handling, and logical
+branch points. To check how much coverage the tests have, you will need.
+
+When running::
+
+    coverage run -m pytest -s --doctest-modules --verbose dwi_ml
+
+You will get the usual output of `pytest`_, but also a table that indicates the
+test coverage in each module: the percentage of coverage and also the lines of
+code that are not run in the tests. You can also see the test coverage in the
+corresponding CI build of the pull request (PR).
+
+If your contributions are to a single module, you can see test and
+coverage results for only that module without running all of the DWI_ML
+tests. For example, if you are adding code to ``dwi_ml/core/utils.py``,
+you can use::
+
+    coverage run --source=dwi_ml.core.utils -m pytest -s --doctest-modules --verbose dwi_ml/core/tests/test_utils.py
+
+You can then use ``coverage report`` to view the results, or use
+``coverage html`` and open ``htmlcov/index.html`` in your browser for a nicely
+formatted interactive coverage report.
+
+Contributions to tests that extend test coverage in older modules that are not
+fully covered are very welcome !
+
+Code style
+----------
+
+Code contributions should be formatted according to the `DWI_ML Coding Style Guideline <./doc/devel/coding_style_guideline.rst>`_
```

Comment:
Missing a point at the end of the line. guideline.rst>`_ .

Reply:
Done in ad7ee3c.